src/views/ViewUtils.ts

```diff
@@ -51,4 +51,15 @@ export class ViewUtils {
     }
     return ColumnDescUtils.numberCol(col, dataSubType.domain[0], dataSubType.domain[1], {label});
   }
+
+  static showMaximumSelectionWarning(selection: ISelection, desc: IViewPluginDesc) {
+    const {name, selectionLimit} = desc;
+    if (selectionLimit && selection.range.size().reduce((a, b) => a + b, 0) > selectionLimit) {
+      NotificationHandler.pushNotification('warning',
+        `<b>${name}</b>: Supported incoming selections limit reached. Showing data for the first <b>${selectionLimit}</b> items.`,
+        NotificationHandler.DEFAULT_SUCCESS_AUTO_HIDE);
```

Comment:
5 seconds are very fast, until I detected the warning and started to read, it was already gone. I suggest to increase the time to 10 seconds or maybe let it close by the user.